tools/crm_attribute.c

```diff
@@ -512,6 +527,8 @@ main(int argc, char **argv)
         if (target != NULL) {
             g_free(options.dest_uname);
             options.dest_uname = g_strdup(target);
+        } else if (getenv("CIB_file") != NULL) {
+            get_node_name_from_local();
```

Comment:
We need to do this only if dest_uname is NULL (i.e. if the user explicitly gave -N we still want to honor that)

Reply:
Fixed.